catalog_tools/analysis/estimate_beta.py

```diff
@@ -4,8 +4,13 @@
 from typing import Optional
 
 
-def estimate_beta_tinti(magnitudes: np.ndarray, mc: float, delta_m: float = 0,
-                        weights: Optional[list] = None) -> float:
+def estimate_beta_tinti(magnitudes: np.ndarray,
+                        mc: float,
+                        delta_m: float = 0,
+                        weights: Optional[list] = None,
+                        gutenberg: bool = False,
+                        error: bool = False
+                        ) -> (float, float):
```

Comment:
@schmidni the output is either just a float or a tuple(float, float), depending on the parameter gutemberg. Is this hinted correctly?

Reply:
Union[float, Tuple[float, float]]?

Reply:
> Union[float, Tuple[float, float]]?

correct.